doc/nrf/libraries/others/network_core_monitor.rst

```diff
@@ -43,13 +39,20 @@ During the startup of the network core, the reset bit is set and the cause of th
 This value is rewritten from the network core's ``RESET.RESETREAS`` register.
 For a detailed description of the bits in this register, see the `RESETREAS`_ section for nRF5340.
 
-The :c:func:`ncm_net_status_check` function returns the following error codes:
+The :c:func:`ncm_net_core_event_handler` function may be implemented by your application.
```

Comment:
IMO it would be good to explain _why_ the user could want to implement this function. For example:
> On the application core, the network core monitor checks the values of IPC registers written by the network core every :kconfig:option:`CONFIG_NCM_FEEDING_INTERVAL_MSEC`. In case the network core malfunctions and fails to increment the ``COUNTER`` value as documented above, :c:func:`ncm_net_core_event_handler` is called on the application core. The network core monitor provides a ``__weak`` implementation of that function...

Without it it's unclear what this function does and why anyone should want to implement it.

Reply:
- In case -> if
- leave out "as documented above"
- The :c:func:... function is called

Reply:
Expanded the description with:

> On the application core, the network core monitor checks the values of IPC registers written by the network core every :kconfig:option:`CONFIG_NCM_FEEDING_INTERVAL_MSEC`.
If the network core malfunctions and fails to increment the ``COUNTER`` value, the :c:func:`ncm_net_core_event_handler` function is called on the application core.
This function is also called when the network core is restarted.
The network core monitor provides a ``__weak`` implementation of that function in the :file:`nrf/subsys/net_core_monitor/app_core.c` file.

The usage section contains a sentence why users may want to implement this function.